rustfmt-core/rustfmt-bin/tests/nested-test-files/empty-tests-dir/Cargo.toml

```diff
@@ -4,3 +4,5 @@ version = "0.1.0"
 authors = ["rustfmt devs <awesome@gmail.com>"]
 
 [dependencies]
+
+[workspace]
```

Comment:
Now that these files are under the workspace, we need to exclude them from the workspace.

cc https://github.com/rust-lang/cargo/issues/6745